doc/source/image_description/elements.rst

```diff
@@ -889,7 +889,15 @@ force_mbr="true|false":
   partitions
 
 gpt_hybrid_mbr="true|false":
-  For GPT disk types only: Create a hybrid GPT/MBR partition table
+  For disk types, create a hybrid GPT/MBR partition table with an
+  'accurate' MBR table that will have no 'bootable' flagged partition
+  For the live ISO type, create a hybrid GPT/MBR partition table
```

Comment:
`live ISO types` -> `ISO types`. This now applies to any ISO created through kiwi.

Reply:
Done.